src/Workspaces/CSharp/Portable/Classification/Worker_DocumentationComments.cs

```diff
@@ -143,7 +145,14 @@ private void ClassifyXmlTextTokens(SyntaxTokenList textTokens)
                 if (token.HasLeadingTrivia)
                     ClassifyXmlTrivia(token.LeadingTrivia);
 
-                ClassifyXmlTextToken(token);
+                if (token.Parent is XmlTextAttributeSyntax { Name.LocalName.Text: DocumentationCommentXmlNames.LangwordAttributeName })
+                {
+                    ClassifyXmlLangwordAttributeValue(token);
+                }
+                else
+                {
+                    ClassifyXmlTextToken(token);
+                }
```

Comment:
📝 This should move up to `ClassifyXmlAttribute`

Reply:
TBO it seems like now code has more repetetive stuff, but the general idea looks good to me though